### DISCUSSION TITLE
Problem accessing documents on mobile ?

### DISCUSSION THREAD

**Discussion by DimitriP-GH** [outline/outline#8162] [Self Hosting]
Hello,

Here's my current setup: a Raspberri Pi 5 (8GB) running a few docker apps on various ports, including one for Outline. All seemed well when I finally got authentication etc working, so since then I've been busy installing/testing other things. Including a Twingate docker that I'll use in the future to access the applications I want on the Rpi when I'm not at home.

A few days ago, I started using Outline, wrote some notes, and tried to access those notes with my phone using Twingate (which acts like a VPN, for simplicity), but without success. Let me explain: when I'm on my mobile, I can see all the documents/collections I've created on my PC, but I can't see anything written inside, at all. And that's the same problem for the opposite, If I write something in it from my phone, I don't see it on my PC. 
The problem also occurs when I'm on the same network with my phone, on Wi-fi and not under my VPN. So I'm guessing it's more a config/error problem, even tho I have no errors in the docker logs of the Outline container. There is just, sometimes, this in logs: ":{"includeChildDocuments":false,"name":"Misc","published":false},". 

Another infos in case it helps: I have 2 different collections with the same document's names inside, but of course a different collection name. I'm using the same auth method and account to login on PC/Mobile, so it shouldn't be a permissions problem since I only have this user. Also, Twingate works by giving access to certain IP/domain with one or multiple ports (TCP/UDP), so I had to use the Outline's IP and port(s) to be able to access it with my phone. Idk, maybe there is another port I have to give access to but that seems weird to be able to see collections/docs names but not what's written in it, just because of a port.

I've searched a bit for similar problems, or thing in the documentation that could create that, but I didn't find anything. Even tho I know I didn't searched enough, being in this holiday/festive period, I'll try my luck here !

**Comment by tommoor** [comment]
It sounds like your documents are not persisting to the server correctly, on self-hosted installations this is usually due to the websockets not correctly connecting – you should look for this and probably adjust your proxy rules. There are many discussions on the topic in here if you search for websockets

**Reply by DimitriP-GH** [reply]
Thanks for the answer, I'll check these discussions and try it out!
